be/src/exec/pipeline/scan/chunk_buffer_limiter.cpp

```diff
@@ -19,7 +20,7 @@ void DynamicChunkBufferLimiter::update_avg_row_bytes(size_t added_sum_row_bytes,
         return;
     }
 
-    size_t chunk_mem_usage = avg_row_bytes * _chunk_size;
+    size_t chunk_mem_usage = avg_row_bytes * max_chunk_rows;
```

Comment:
Maybe should record a `_max_chunk_rows` for the maximum historical chunk_rows.

Reply:
Later.